### PR TITLE
Added Omicron support to the auto generate list

### PR DESCRIPTION
**Pull request by NducTiOnomBi**
Added Omicron boost support along with persisting player selections
Updated Ewok targets (Thought I had submitted this previously)
Updated characters than deal special and mixed damage
Added Mixed and Special damage OptimizationPlans

### CODE REVIEW

File: src/containers/CharacterEditView/CharacterEditView.js [hunk 1]
```diff
@@ -470,12 +476,45 @@ class CharacterEditView extends PureComponent {
...
         <label htmlFor={'use-case'}>Select your use case:</label>
-        <Dropdown name={'use-case'}>
-          <option value={''}>Grand Arena / Territory Wars</option>
+        <Dropdown id={'use-case'} name={'use-case'}>
+          <option value={''}>GAC / TW / ROTE (default)</option>
           <option value={1}>Light-side Territory Battle</option>
           <option value={2}>Dark-side Territory Battle</option>
           <option value={3}>Arena only</option>
         </Dropdown>
+        <p/>
```
Comment (grandivory) [inline src/containers/CharacterEditView/CharacterEditView.js:485]: This is a little odd. What's this empty paragraph doing here? Should it be a line break, or a horizontal break?

File: src/containers/CharacterEditView/CharacterEditView.js [hunk 2]
```diff
@@ -531,11 +570,15 @@ class CharacterEditView extends PureComponent {
           </div> */}
           <div className={'form-row'}>
             <label htmlFor={'ignore-arena'}>Ignore arena teams:</label>
-            <input name={'ignore-arena'} type={'checkbox'} defaultChecked={true} />
+            <input id={'ignore-arena'} name={'ignore-arena'} type={'checkbox'} defaultChecked={true} />
           </div>
+          {/* <div className={'form-row'}>
+            <label htmlFor={'ignore-synergy'}>Ignore synergy:</label>
+            <input id={'ignore-synergy'} name={'ignore-synergy'} type={'checkbox'} defaultChecked={false} />
+          </div> */}
```
Comment (grandivory) [inline src/containers/CharacterEditView/CharacterEditView.js:578]: If this isn't intended to be used, mind removing it?

File: src/containers/CharacterEditView/CharacterEditView.js [hunk 3]
```diff
@@ -549,7 +592,14 @@ class CharacterEditView extends PureComponent {
               'alignmentFilter': form['alignment-filter'].value,
               'minimumGearLevel': form['minimum-gear-level'].value,
               'ignoreArena': form['ignore-arena'].checked,
-              'top': form['max-list-size'].value
+              //'gearLevelThreshold': form['gear-level-threshold'].value,
+              //'ignoreSynergy': form['ignore-synergy'].checked,
```
Comment (grandivory) [inline src/containers/CharacterEditView/CharacterEditView.js:596]: Same here

**Review by grandivory — COMMENTED** [review]
Just a few small things to clean up. Thanks for all the work setting the right damage types in character settings!

File: src/containers/CharacterEditView/CharacterEditView.js (hunk 1 above)
Comment (NducTiOnomBi) [reply src/containers/CharacterEditView/CharacterEditView.js:485]: Oops! Meant to replace with some css padding. Thanks

File: src/containers/CharacterEditView/CharacterEditView.js (hunk 2 above)
Comment (NducTiOnomBi) [reply src/containers/CharacterEditView/CharacterEditView.js:578]: Removing

File: src/containers/CharacterEditView/CharacterEditView.js (hunk 3 above)
Comment (NducTiOnomBi) [reply src/containers/CharacterEditView/CharacterEditView.js:596]: Removing

**Review by grandivory — APPROVED** [review]

File: src/containers/CharacterEditView/CharacterEditView.js (hunk 1 above)
Comment (NducTiOnomBi) [reply src/containers/CharacterEditView/CharacterEditView.js]: Fixed

File: src/containers/CharacterEditView/CharacterEditView.js (hunk 2 above)
Comment (NducTiOnomBi) [reply src/containers/CharacterEditView/CharacterEditView.js]: Removed

File: src/containers/CharacterEditView/CharacterEditView.js (hunk 3 above)
Comment (NducTiOnomBi) [reply src/containers/CharacterEditView/CharacterEditView.js]: Removed

**Pull request merged**